.github/CODEOWNERS

```
@@ -2,21 +2,132 @@
 # Each line is a file pattern followed by one or more owners.
 # https://help.github.com/articles/about-codeowners/
 
-*git*/* @sourcegraph/core-services
+# Top-level catch-alls (these are weaker confidence and might need to be reassigned at some point)
+/enterprise/cmd/frontend @beyang
+/enterprise/cmd/server @beyang
+/enterprise/dev @beyang
+/cmd/frontend/docsite @sqs
+/cmd/frontend/shared @beyang
+/cmd/frontend/backend @beyang
+/cmd/frontend/internal/app/assets @beyang
+/cmd/frontend/internal/app/templates @beyang
+/cmd/frontend/internal/app/canonicalurl @beyang
+/cmd/frontend/internal/app/*.go @beyang
+/cmd/frontend/internal/app/assetsutil @beyang
+/cmd/frontend/internal/app/ui @beyang
+/cmd/frontend/internal/app/returnto @beyang
+/cmd/frontend/internal/app/pkg @beyang
+/cmd/frontend/internal/app/router @beyang
+/cmd/frontend/internal/app/errorutil @beyang
+/cmd/frontend/internal/goroutine @keegancsmith
+/cmd/frontend/internal/inventory @beyang
+/cmd/frontend/internal/cli/middleware @beyang
+/cmd/frontend/internal/cli @beyang
+/cmd/frontend/internal/pkg/siteid @beyang
+/cmd/frontend/internal/pkg/suspiciousnames @beyang
+/cmd/frontend/internal/pkg/markdown @beyang
+/cmd/frontend/internal/pkg/handlerutil @beyang
+/cmd/frontend/internal/httpapi @beyang
+/cmd/frontend/types @beyang
+/cmd/frontend/hooks @beyang
+/cmd/frontend/internal/ @beyang
+/pkg/randstring/ @beyang
+/pkg/pubsub/ @beyang
+/pkg/repotrackutil/ @beyang
+/pkg/atomicvalue/ @beyang
+/pkg/testutil/ @beyang
+/pkg/debugserver/ @beyang
+/pkg/vfsutil/ @beyang
+/pkg/gituri/ @beyang
+/pkg/db/ @keegancsmith
+/pkg/processrestart @keegancsmith
+/pkg/honey @keegancsmith
+/pkg/ratelimit @beyang
+/pkg/registry @chrismwendt
+/pkg/slack @beyang
+/pkg/prefixsuffixsaver @beyang
+/pkg/gosrc @beyang
+/pkg/txemail @beyang
+/gulpfile.ts @beyang
+/client @felixfbecker
```

```suggestion
/client @sqs 
```
This is only the bookmarklet